include/fastgltf/types.hpp

```diff
@@ -1502,6 +1502,19 @@ namespace fastgltf {
         Optional<TextureInfo> sheenRoughnessTexture;
     };
 
+#if FASTGLTF_ENABLE_DEPRECATED_EXT
+    /**
+     * Specular/Glossiness information from KHR_materials_pbrSpecularGlossiness.
+     */
+    struct MaterialSpecularGlossiness {
+        std::array<float, 4> diffuseFactor;
+        std::optional<TextureInfo> diffuseTexture;
+        std::array<float, 3> specularFactor;
+        float glossinessFactor;
+        std::optional<TextureInfo> specularGlossinessTexture;
```

Comment:
Replace std::optional with Optional. Its a type alias that checks wether a specialisation for fastgltf::OptionalWithFlagValue exists for T. When I do specialise it in the future for TextureInfo, I wouldn't have to change these lines.

Reply:
Done